lib/tangocard.rb

```diff
@@ -25,6 +31,14 @@ def self.configuration
 
   def self.configure
     yield(configuration) if block_given?
+    warm_cache if configuration.use_cache
+  end
+
+  def self.warm_cache
+    configuration.cache.write("#{Tangocard::CACHE_PREFIX}rewards_index", Tangocard::Raas.rewards_index(bypass_cache: true))
+    configuration.logger.info('Warmed Tangocard cache')
+  rescue Exception => e
```

Comment:
Not sure what they are. For now I'll leave off the rescue and see if we get any Airbrakes.